extension/xextension/storage/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+type: xextension
+github_project: open-telemetry/opentelemetry-collector
+
+status:
+  class: extension
+  codeowners:
+    active:
+      - swiatekm
+  stability:
+    development: [profiles]
```

Comment:
Does storage has any dependency on profiles?

Reply:
not yet but I guess that's coming, at least that's what that bug was about.